data/net/minecraft/world/level/material/Fluid.mapping

```diff
@@ -20,6 +20,18 @@ CLASS net/minecraft/world/level/material/Fluid
 		ARG 1 blockReader
 		ARG 2 pos
 		ARG 3 fluidState
+	METHOD getHeight (Lnet/minecraft/world/level/material/FluidState;Lnet/minecraft/world/level/BlockGetter;Lnet/minecraft/core/BlockPos;)F
+		ARG 1 fluidState
+		ARG 2 blockGetter
```

Comment:
```suggestion
		ARG 1 state
		ARG 2 level
```
While this class is a bit of a mess, using both `fluidState` and `state` as parameters in different places, given this class (`Fluid` / `FluidState`) is akin to `Block` / `BlockState`, unless there's an explicit block state *and* fluid state mentioned, I think these methods make more sense to have just a `state` parameter. The `fluid` part is implicit.

If you wanted to do a consistency pass over this class and change the existing `fluidState` to `state`, I'd appreciate that.

Reply:
Will do